content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,155 @@
+
+This model has the following advantages:
+
+* Reduces the blast radius of bad configuration and helps to maintain high availability for the control plane.
+* Allows administrators more fine-grained control over configuration rollout (e.g. one cluster at a time). For large
+meshes with many clusters, this process can be automated with CI/CD pipelines.
+* Allows selective exposure of `Service`/`ServiceEntry` across cluster boundaries. For example, an administrator may
+choose to deploy the HelloWorld service to Cluster A, but not Cluster B. This is an advanced use case, but may be
+desirable for stronger isolation between clusters.
+
+This duplication of configuration across primary clusters can lead to issues, however. In particular, Kubernetes
+Services must be deployed to each cluster that {{< gloss "service consumer">}}consumes{{</ gloss >}} the service
+in order to allow DNS lookup to succeed. See [DNS and Service Deployment](#dns-and-service-deployment) for more
+information.
+
+### Endpoint Discovery
+
+An Istio control plane manages traffic within the mesh by providing each proxy with the list of service endpoints.
+In order to make this work in a multicluster scenario, each control plane must observe endpoints from the API Server
+in every cluster.
+
+To enable endpoint discovery across the mesh:
+
+* Create a remote secret for each cluster, which grants access to its API server.
+* Deploy each remote secret to each primary cluster (within the Istio system namespace).
+
+The control planes will then connect and discover the service endpoints from each cluster, enabling cross-cluster
+load balancing for these services.
+
+{{< image width="80%"
+    link="endpoint-discovery.svg"
+    caption="Primary clusters with endpoint discovery"
+    >}}
+
+By default, Istio will load balance requests evenly between endpoints in each cluster. In large systems that span
+geographic regions, it may be desirable to use locality load balancing to prefer that traffic stay in the same zone
+or region.
+
+In some advanced scenarios, load balancing across clusters may not be desired. For example, in a blue/green
+deployment, you may deploy different versions of the system to different clusters. In this case, each cluster is
+effectively operating as an independent mesh. This behavior can be achieved in a couple of ways:
+
+* Do not exchange remote secrets between the clusters. This offers the strongest isolation between the clusters.
+* Use `VirtualService` and `DestinationRule` to disallow routing between two versions of the services.
+
+In either case, cross-cluster load balancing is prevented. External traffic can be routed to one cluster or the
+other using an external load balancer.
+
+{{< image width="80%"
+    link="blue-green.svg"
+    caption="Blue-green deployment without cross-cluster load balancing"
+    >}}
+
+### Remote Clusters
+
+A {{< gloss >}}remote cluster{{< /gloss >}} refers to a cluster that contains user workloads but no control plane.
```

Comment:
Is this doc only valid once we get istiod-less remotes? Or we need to clarify that "control plane" in this context only refers to serving xDS, but not Certs/Injection.

Reply:
That was my thought. I'm trying to be generic here, so that we don't fumble around to much WRT "types" of remote clusters. So yeah, I'm considering the source of XDS as the control plane in this case.